core/state_processor.go

```diff
@@ -1141,7 +1141,7 @@ func (p *StateProcessor) GetKQuaiAndUpdateBit(parent *types.WorkObject) (*big.In
 	}
 	state, err := p.StateAt(evmRoot, etxRoot, quaiStateSize)
 	if err != nil {
-		return nil, 0, err
+		return nil, 0, ErrSubNotSyncedToDom
```

Comment:
I don't think it's a good idea to throw away the error here, there are many different errors that could be returned from this function

Reply:
I will log the error, but this is necessary because we will cause the append to fail as prime is trying to lookup state on a parent zone block and in a network setting there can be a delay and we cannot fail the append because of it, If state is available we cannot proceed appending that prime block ( either it runs out of tries or gets appended once the parent zone state is ready)